blogposts/2021/dev-tool-time-mitchell-hashimoto.md

```diff
@@ -1,17 +1,17 @@
 ---
 title: "'I don't like things that write code for me': Dev Tool Time with Mitchell Hashimoto"
-description: 'Mitchell Hashimoto, co-founder, CTO, and now individual contributor at HashiCorp, shares how reproducible environments with NixOS and an iPad fit into his developer workflow.'
+description: 'Mitchell Hashimoto, co-founder, CTO, and now individual contributor at HashiCorp, shares how reproducible environments with NixOS and an iPad fit into his developer workflow, and his sentiments about IDEs.'
```

Comment:
@csbailey5t I think they spend enough time on the Vim conversation that the quote works, I just adjusted the description to reference it too though